server/common/transport-shared/src/main/java/org/kaaproject/kaa/server/transport/channel/ChannelType.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2014 CyberVision, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kaaproject.kaa.server.transport.channel;
+
+/**
+ * Represents types of channels supported by Kaa server components. Messages
+ * that are dispatched by specific channel may be handled differently based on
+ * channel type.
+ * 
+ * @author Andrew Shvayka
+ *
+ */
+public enum ChannelType {
+
+    /**
+     * Sync channel identifies that messages from this channel require immediate
```

Comment:
The sync channel identifies that messages from this channel require an immediate reply. One of the sync channels is a regular http channel.